specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/application/rfc7540/connect/client.rpt

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2021-2022 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "zilla://streams/app0"
+        option zilla:window 8192
+        option zilla:transmission "half-duplex"
+
+
+write zilla:begin.ext ${http:beginEx()
+                        .typeId(zilla:id("http"))
+                        .header(":scheme", "http")
+                        .header(":method", "CONNECT")
+                        .header(":protocol", "websocket")
+                        .header(":path", "/chat")
+                        .header(":authority", "localhost:8080")
+                        .build()}
+connected
+
+read zilla:begin.ext ${http:beginEx()
+                           .typeId(zilla:id("http"))
+                           .header(":status", "200")
+                           .header("server", "CERN/3.0 libwww/2.17")
+                           .header("date", "Wed, 01 Feb 2017 19:12:46 GMT")
+                           .build()}
```

Comment:
Use `http:matchBeginEx` when reading, and only include the headers that matter per spec, such as `:status` above.